packages/openassistants/openassistants/contrib/text_response.py

```diff
@@ -0,0 +1,53 @@
+from typing import Annotated, Any, List, Literal, Sequence, Optional,Callable,Dict
+import os
+import inspect
+import pandas as pd
+
+from openassistants.data_models.function_input import BaseJSONSchema
+from openassistants.data_models.function_output import FunctionOutput,TextOutput,SuggestedPrompt,FollowUpsOutput
+from openassistants.functions.base import BaseFunction, FunctionExecutionDependency
+from openassistants.functions.utils import AsyncStreamVersion
+from openassistants.utils.strings import resolve_str_template
+from pydantic import Field
+
+from langchain.chat_models import ChatOpenAI
+from langchain.schema.messages import HumanMessage, SystemMessage
+import base64
+
+
+class TextResponseFunction(BaseFunction):
+    type: Literal["TextResponseFunction"] = "TextRespnseFunction"
+    parameters: BaseJSONSchema
```

Comment:
I think we should get rid of `parameters` and `get_parameters_json_schema`. They're not longer necessary and not used by this class.